Removing DockerCli.pullImage as it's never used

vdemeester opened the pull request:
This function is never used nor exported. It has been superseeded by `pullImageCustomOut` 🐷.

🐸

Signed-off-by: Vincent Demeester vincent@sbr.pm

aaronlehmann commented:
LGTM, but can you rename `pullImageCustomOut` to `pullImage` to simplify the naming?

vdemeester commented:
@aaronlehmann Yes I can :)

vdemeester committed:
Removing DockerCli.pullImage as it's never used

This function is never used nor exported. It has been superseeded by
pullImageCustomOut.

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

LK4D4 commented:
LGTM

icecrime commented:
userns failure fixed in #21514.
